### DISCUSSION TITLE
Please allow admins to tp to offline players beds

### DISCUSSION THREAD

**Discussion by Klawdek** [EssentialsX/Essentials#5196] [Ideas and Feature Suggestions]
This is about TPing to offline players beds. This has NOTHING to do with TPing offline players.  Please do not start talking about TPing offline players anywhere. 

It would be nice if we could TP to offline players beds.  I see no reason why that should not be allowed at least for admins.  Yes it would have to have perms like other commands and could be allowed for admins and select players.

**Comment by baailey** [comment]
Can't you already do that using `/home username:bed`? Or does that not work?
